docs/recipes/client-server/fable.forms.md

```diff
@@ -0,0 +1,237 @@
+## Install dependencies
+
+First off, you need to create a SAFE app, [install the relevant dependencies](https://mangelmaxime.github.io/Fable.Form/Fable.Form.Simple.Bulma/installation.html), and wire them up to be available for use in your F# Fable code.
+
+1. Create a new SAFE app and restore local tools:
+```sh
+dotnet new SAFE
+dotnet tool restore
+```
+1. Add bulma to your project:
+follow [this recipe](../ui/add-bulma.md)
+
+1. Install Fable.Form.Simple.Bulma using Paket:
+```sh
+dotnet paket add Fable.Form.Simple.Bulma -p Client
+```
+
+1. Install bulma and fable-form-simple-bulma npm packages:
+```sh
+npm add fable-form-simple-bulma
+npm add bulma
+```
+
+## Register styles
+
+1. Rename `src/Client/Index.css` to `Index.scss`
+
+2. Update the import in `App.fs`
+
+    === "Code"    
+        ```.fsharp title="App.fs"
+        ...
+        importSideEffects "./index.scss"
+        ...
+        ```
+    === "Diff"    
+        ```.diff title="App.fs" 
+        ...
+        - importSideEffects "./index.css"
+        + importSideEffects "./index.scss"
+        ...
+        ```
+
+3. Import bulma and fable-form-simple in `Index.scss`
+
+    ``` .scss title="Index.scss"
+    @import "bulma/bulma.sass";
+    @import "fable-form-simple-bulma/index.scss";
+    ...
```

Comment:
I realized when implementing your feedback that I made a mistake here... Vite does not deal well with the  "~" imports, so we need to use this notation